platforms/ios/framework/src/TGMapView.mm

```diff
@@ -38,6 +38,15 @@ inline CLLocationDirection convertRotationRadiansToBearingDegrees(float rotation
     return TGDegreesFromRadians(-rotation);
 }
 
+/**
+ Map region change states
+ */
+typedef NS_ENUM(NSInteger, TGMapRegionChangeStates) {
```

Comment:
Conventionally, enum names are singular - it makes a little more sense when using it as a type name.